ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoModule.java

```diff
@@ -9,24 +9,55 @@
 
 package com.facebook.react.modules.systeminfo;
 
+import android.app.UiModeManager;
+import android.content.res.Configuration;
 import android.os.Build;
 
-import com.facebook.react.bridge.BaseJavaModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.module.annotations.ReactModule;
 
 import java.util.HashMap;
 import java.util.Map;
 
 import javax.annotation.Nullable;
 
+import static android.content.Context.UI_MODE_SERVICE;
+
 /**
  * Module that exposes Android Constants to JS.
  */
 @ReactModule(name = "PlatformConstants")
-public class AndroidInfoModule extends BaseJavaModule {
+public class AndroidInfoModule extends ReactContextBaseJavaModule {
 
   private static final String IS_TESTING = "IS_TESTING";
 
+  private ReactContext mReactContext;
```

Comment:
When extending `ContextBaseJavaModule` I think you don't need to keep reference to the context yourselves – you should be able to call `getReactApplicationContext()` to get reference to the context